### ISSUE TITLE
Questions about Intra- and Inter-Action Understanding via Temporal Action Parsing

### ISSUE THREAD

**Issue by KevinQian97** [SDOlivia/FineGym#3]
Hi, congratulates on your publication in CVPR 2020.
I am interested in your paper "Questions about Intra- and Inter-Action Understanding via Temporal Action Parsing" and get directed here from https://sdolivia.github.io/TAPOS/.
I have several questions:
1) Do you have a released repo for Intra- and Inter-Action Understanding via Temporal Action Parsing?
2) You report the results on the test set or the validation set? Cuz I only find train/val splits in your released annotation file. Please correct me if I am wrong.
3) For results in Table 1, would u mind a bit more explanation about "segments aligned with annotated sub-actions"? Presently I think it means align different frames for each sub-actions according to their length. Then randomly sample corresponding frames from these sub-actions and concatenate together to generate the whole input clip to TSN. However, it seems to bring no difference compared with random sampling.
Thanks!
